scripts/test-ssr/src/utils/visitPage.ts

```diff
@@ -24,20 +26,19 @@ export async function visitPage(browser: Browser, url: string) {
         return;
       }
 
-      // Ignoring network errors as we have an interceptor that prevents loading everything except our JS bundle
-      if (!messageContent.includes('net::ERR_FAILED')) {
-        error = new RenderError(messageContent);
-      }
+      error = new RenderError(messageContent);
     }
   });
 
   page.on('request', request => {
-    // Our interceptor allows only our HTML and JS output
-    if (request.url() === url || request.url().endsWith('/out-esm.js')) {
-      return request.continue();
+    if (match.isMatch(request.url(), ['**/*.png', '**/*.jpg', '**/*.jpeg', '**/*.gif', '**/*.svg'])) {
+      return request.respond({
+        status: 200,
+        body: '',
+      });
     }
 
-    return request.abort();
+    return request.continue();
```

Comment:
TIL how to do it properly

Reply:
@Hotell sure 👍 TL;DR We don't want to load any images from CDN to avoid timeouts and make tests faster.

Before: Allow to load `index.html` & `dist/out-esm.js`, for other assets abort a request, have to handle "net::ERR_FAILED" in console 👎 
After: Requests to images are responded with HTTP 200 and empty body, no errors 👍